app/views/research_sessions/preview.html.erb

```diff
@@ -64,16 +64,13 @@
   <% end %>
 
   <% recording = capture do %>
-    <p>Which we will record using:</p>
-    <ul class="bullet-point-list">
-      <% @research_session.recording_methods.each do |recording_method| %>
-        <li>
-          <%= edit_link_for(:recording_methods) do %>
-            <%= recording_method_lookup(recording_method) %>
-          <% end %>
-        </li>
-      <% end %>
-    </ul>
+    <%=
+      react_component(
+        'Shared/WhichWeWillRecordUsingList',
+        preview_params(:recording).merge(all_recording_methods: RecordingMethods::NAME_VALUES),
+        { prerender: true }
+      )
```

Comment:
This is still in a capture block; should really go into `WhatHappensInThisResearchSession` below, because we are feeding that component a bunch of dangerous markup.

Reply:
We need a card for this @rgarner